BonusCalcApi/V1/Infrastructure/Migrations/20230406141810_AddSpArchiveOperative.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace V1.Infrastructure.Migrations
+{
+    public partial class AddSpArchiveOperative : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(
+@"
+CREATE OR REPLACE PROCEDURE public.sp_archive_operative(
+	op_payroll_number text,
+	op_leave_year text)
+LANGUAGE 'sql'
+AS $BODY$
+UPDATE public.operatives SET is_archived = TRUE,
+""name"" = name ||' LEFT '||op_leave_year
```

Comment:
Should it be '"name"' or does it work with double double quotes anyway?
Also where does 'name' (the target value) come from?

Reply:
"" (double double quotes) is the C# notation for a double quote literal (")
It's the same column as the assignment - it's basically saying "Set name to _name LEFT 2023_"